instruqt/avoiding-installation-pitfalls/01-working-with-preflight-checks/assignment.md

```diff
@@ -0,0 +1,145 @@
+---
+slug: working-with-preflight-checks
+id: 2pg7yavlxy31
+type: challenge
+title: Working with Preflight Checks
+teaser: |-
+  Learn how preflight checks help you avoid common install
+  mistakes
+notes:
+- type: text
+  contents: Let's learn about Preflights
+tabs:
+- title: Shell
+  type: terminal
+  hostname: shell
+- title: Manifest Editor
+  type: code
+  hostname: shell
+  path: /home/replicant
+difficulty: basic
+timelimit: 300
+---
+
+👋 Introduction
+===============
+
+One of the areas teams worry about most when distributing software
+to customer clusters is whether those clusters will be properly
+configured for the application to run. To address this challenge,
+the Replicated Platform provides for the implementation and
+management for preflight checks that can describe your expectations
+and validate them. Preflights are part of the
+[Troubleshoot](https://troubleshoot.sh) open source project.
+
+ What is a Preflight Check?
+===========================
+
+Preflight checks analyze your customer's cluster as a client without
+installing anything into it. This allows customers to run Preflight
+checks at any time before installing your software and keep the
+cluster unchanged. They codify the pre-requisites you would catalog
+in your documentation and make the executable.
+
+You define your checks in a YAML file that follows the same format
+as a Kubernetes object. The simplest preflight object looks like this,
+and it's in the file `empty-preflights.yaml`:
```

Comment:
the file in the lab is named `emtpy-preflight.yaml` so the command doesn't work

Reply:
Updating to `empty-preflights.yaml`